disperser/controller/dispatcher_metrics.go

```diff
@@ -181,6 +191,31 @@ func newDispatcherMetrics(registry *prometheus.Registry) *dispatcherMetrics {
 		"send_to_validator",
 		false)
 
+	// Verify that thresholds are sane
+	for _, threshold := range importantSigningThresholds {
+		if threshold < 0 || threshold > 1 {
+			return nil, fmt.Errorf("threshold %f is not between 0.0 and 1.0", threshold)
+		}
+	}
+	sort.Float64s(importantSigningThresholds)
+
+	// Add thresholds for 0.0 and 1.0, if missing.
+	if len(importantSigningThresholds) == 0 || importantSigningThresholds[0] != 0.0 {
+		importantSigningThresholds = append([]float64{0.0}, importantSigningThresholds...)
+	}
+	if importantSigningThresholds[0] != 1.0 {
```

Comment:
Is this the correct condition? Shouldn't it be:

```
if importantSigningThresholds[len(importantSigningThresholds) -1] != 1.0 {
```
?

Reply:
You are correct. Fixed.

The net effect of this bug would have been for all 100% signed batches to be counted in the 0.67 bucket.